### PR TITLE
update version 0.5.1

### PR DESCRIPTION
**Pull request by germainruget**

### CODE REVIEW

File: template-share-extension/package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "@lumx/react": "^2.0.1",
     "axios": "0.21.1",
     "lodash": "4.17.21",
-    "lumapps-sdk-js": "^1.0.1",
+    "lumapps-sdk-js": "^1.0.0",
```
Comment (LaurentMox) [inline template-share-extension/package.json:43]: This version is bugged, please use the previous one (0.0.44 as far as i remember)

File: template-widget-extension/package.json [hunk 2]
```diff
@@ -1,91 +1,91 @@
...
+    ]
+  },
+  "dependencies": {
+    "@lumapps-extensions-playground/common": "^1.3.0",
+    "@lumx/core": "^2.0.1",
+    "@lumx/icons": "^2.0.1",
+    "@lumx/react": "^2.0.1",
+    "axios": "0.21.1",
+    "lodash": "4.17.21",
+    "lumapps-sdk-js": "^1.0.0",
```
Comment (LaurentMox) [inline template-widget-extension/package.json:43]: This version is bugged, please use the previous one (0.0.44 as far as i remember)

**Review by LaurentMox — CHANGES_REQUESTED** [review]

**Review by gregorycat — APPROVED** [review]

File: template-widget-extension/package.json [hunk 3]
```diff
@@ -1,91 +1,91 @@
...
+    "transformIgnorePatterns": [
+      "node_modules/(?!@lumx/icons|@lumx/react)"
+    ]
+  },
+  "dependencies": {
+    "@lumapps-extensions-playground/common": "^1.3.0",
+    "@lumx/core": "^2.0.1",
+    "@lumx/icons": "^2.0.1",
+    "@lumx/react": "^2.0.1",
+    "axios": "0.21.1",
```
Comment (LaurentMox) [inline template-widget-extension/package.json:41]: On the _web_ they updagraded to 0.21.2, can you do it too please?

**Review by LaurentMox — APPROVED** [review]
LGTM

File: template-widget-extension/package.json (hunk 3 above)
Comment (LaurentMox) [reply template-widget-extension/package.json:41]: To 0.21.4 sorry, it's the latest patch

**Review by gregorycat — APPROVED** [review]

**Review by LaurentMox — APPROVED** [review]

**Pull request merged**